pkgs/games/openra-mods/mods.nix

```diff
@@ -0,0 +1,293 @@
+{ fetchFromGitHub }:
+
+let
+  extraPostFetch = ''
+    sed -i 's/curl/curl --insecure/g' $out/thirdparty/{fetch-thirdparty-deps,noget}.sh
+    $out/thirdparty/fetch-thirdparty-deps.sh
+  '';
+
+in [
+  {
```

Comment:
In nixpkgs a more idiomatic approach would be to introduce a function like `buildOpenRAMod` (come up with a better name) and then to have attribute set:

```nix
{
  inherit buildOpenRAMod;

  redalert2 = buildOpenRAMod {
    version = "123";
    engine = {
      # ...
    };
    # ...
  };

  # ...
}
```
Then `buildOpenRAMod` can be reused by external mods.